AppCenterDistribute/AppCenterDistribute/MSACDistribute.h

```diff
@@ -65,14 +66,14 @@ typedef NS_ENUM(NSInteger, MSACUpdateTrack) {
  *
  * @param apiUrl The new URL.
  */
-+ (void)setApiUrl:(NSString *)apiUrl;
++ (void)setApiUrl:(NSString *)apiUrl NS_SWIFT_NAME(set(apiUrl:));
```

Comment:
Convert to property.
Also, please look at "Compensate for weak type information" rule https://swift.org/documentation/api-design-guidelines/